### PR TITLE
Multiple NIC integration test

### PR DESCRIPTION
**Pull request by kokesak**
Add an integration test that verifies it is possible to do a full conversion of a system with multiple NICs.

Due to hardware requirements, we need to run this test on Beaker.

Jira Issues: [RHELC-1192](https://issues.redhat.com/browse/RHELC-1192)

### CODE REVIEW

**Comment by kokesak** [comment]
There is some issue when listing tmt tests.

Running the `multiple-nic` test manually  is ok:
c7: http://artifacts.osci.redhat.com/testing-farm/3de4f91d-a29e-4fa8-9bf3-dd4f678c1053
c8: http://artifacts.osci.redhat.com/testing-farm/0a30cf20-1e37-46f6-91eb-d03365d6d4ec

I'm gonna take a look on why it fails here

**Comment by kokesak** [comment]
> There is some issue when listing tmt tests.
> 
> Running the `multiple-nic` test manually is ok: c7: http://artifacts.osci.redhat.com/testing-farm/3de4f91d-a29e-4fa8-9bf3-dd4f678c1053 c8: http://artifacts.osci.redhat.com/testing-farm/0a30cf20-1e37-46f6-91eb-d03365d6d4ec
> 
> I'm gonna take a look on why it fails here

The fix: https://github.com/oamg/convert2rhel/pull/971

File: tests/integration/tier1/destructive/multiple-nic/main.fmf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+summary: Convert a system when having multiple NIC
+
+description: |
+  Regular full conversion of the system that has more than one NIC connected and running.
+
+test: |
+  pytest -svv
```
Comment (danmyway) [inline tests/integration/tier1/destructive/multiple-nic/main.fmf:7]:
```suggestion
summary+: |
  Conversion with multiple NICs

description+: |
  Verify, that it's possible to do a full conversion of the system 
  that has more than one NIC connected and running.
  The test uses RHSM as a conversion method.

test: |
  pytest -svv
```

**Review by danmyway — CHANGES_REQUESTED** [review]
Just some metadata rephrasing, other than that looks good! 
Thanks!

File: tests/integration/tier1/destructive/multiple-nic/main.fmf (hunk 1 above)
Comment (kokesak) [reply tests/integration/tier1/destructive/multiple-nic/main.fmf]: Thanks, I changed it.

**Review by danmyway — APPROVED** [review]
LGTM, thanks!

**Pull request merged**